### PR TITLE
Implement GUI for API keys

### PR DESCRIPTION
**Pull request by blokhin**

### CODE REVIEW

File: src/services/api.ts [hunk 1]
```diff
@@ -120,6 +121,17 @@ export async function getUsers(ids: number[] = []): Promise<UserDTO[]> {
 	return getJSON('/users', { ids: ids.join(',') });
 }
 
+export async function manageAPIKey(action: string): Promise<void> {
```
Comment (knopki) [inline src/services/api.ts:124]: I don't like magic string in the argument. @pyoner maybe enum/union of strings/symbols is better?

Comment (pyoner) [reply src/services/api.ts:124]: @knopki I think union of string literal is enough
enum generate additional code, [see](https://www.typescriptlang.org/play?#code/PTAEFUDsEsHtIMoBcBO1IHMAy0kFMUBDAG1AFgAoJATwAc8IZ5k1Md8jSBeUAcgDNYsXqAA+fAEaEUI8bykAvXgG5KlEKACikAK4BbUIcp5dB7ftABvSodAAxAPIPQPAUN4AaG4YBCAQQAlF0lpT29QfwAtYPlCJUoAXzUKDQBheABnJHMDIwoAY0ykUBMLdMgsnKtwx2dXQWEvClt-INcpGSaWv2j2uN5E5MoAN2lQQgAuRjhEVHRsXAISYIAiDpWRsYkpqp4cgDpWzZRQfKnyytNgi+zTQ8DkoA)

File: src/stores/apikey.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+		} catch(e) {
+			error.set(e);
+		}
+		loading.set(false);
+	}
+
+	controlAPIKey('get');
+
+	return [apikey, loading, error, controlAPIKey];
```
Comment (pyoner) [inline src/stores/apikey.ts:26]: @blokhin I don't like this idea
look at my idea
```typescript
const apiStore = writable<Promise<string>>(...)

// create api key
apiStore.set(apiKey) or apiStore.set({action: 'set', value: apiKey})

// delete api key
apiStore.delete() or apiSore.set({action: 'delete'})
```

read api key and show errors
```svelte
{#await $apiStore}
... waiting api key
{:then apiKey}
... show api key
{:catch error}
... show error
{/await}
```

Comment (blokhin) [reply src/stores/apikey.ts:26]: @pyoner this is not really obvious for me :laughing: actually there is `asyncable` store from @PaulMaly which might be even more appropriate here, but I still haven't got a chance to wrap my head around it...

Comment (pyoner) [reply src/stores/apikey.ts:26]: @blokhin look at [this](https://svelte.dev/repl/7f87e103dcf84662a6ad6cbdee3c99b7?version=3.59.1)

Comment (blokhin) [reply src/stores/apikey.ts:26]: @pyoner OK, I see your approach and have taken the arrow function expressions from it. Its only drawback is that the `key` variable is imported from the store implicitly, which is bad. However let's not spend more time on refactoring this.

**Pull request merged**